src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,698 @@
+
+Si vous voulez éviter les bibliothèques tierces, [Svelte stores](https://svelte.dev/tutorial/writable-stores) sont à eux seuls un excellent outil de communication inter-îles. Néanmoins, vous préférerez peut-être Nano Stores si a) vous aimez leurs modules complémentaires pour ["objects"](https://github.com/nanostores/nanostores#maps) et [async state](https://github.com/nanostores/nanostores#lazy-stores), ou b) vous souhaitez communiquer entre Svelte et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
+</details>
+
+<details>
+<summary>**🙋 Comment Solid signals est-il différent de Nano Stores?**</summary>
+
+Si vous utilisez Solid depuis un certain temps, vous avez peut-être essayé de déplacer [signals](https://www.solidjs.com/docs/latest#createsignal) ou [stores](https://www.solidjs.com/docs/latest#createstore) en dehors de vos composants. C'est un excellent moyen de partager l'état entre les Solid Islands ! Essayez d'exporter des signaux à partir d'un fichier partagé:
+
+```js
+// sharedStore.js
+import { createSignal } from 'solid-js';
+
+export const sharedCount = createSignal(0);
+```
+...et tous les composants important `sharedCount` partageront le même état. Bien que cela fonctionne bien, vous préférerez peut-être Nano Stores si a) vous aimez leurs modules complémentaires pour
+ ["objects"](https://github.com/nanostores/nanostores#maps) et [async state](https://github.com/nanostores/nanostores#lazy-stores), ou b) vous souhaitez communiquer entre Solid et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
+
+</details>
+:::
+
+## Installer Nano Stores
+
+Pour commencer, installez Nano Stores avec leur package d'assistance pour votre framework d'interface utilisateur préféré:
+
+<UIFrameworkTabs>
+  <Fragment slot="preact">
+  ```shell
+  npm install nanostores @nanostores/preact
+  ```
+  </Fragment>
+  <Fragment slot="react">
+  ```shell
+  npm install nanostores @nanostores/react
+  ```
+  </Fragment>
+  <Fragment slot="solid">
+  ```shell
+  npm install nanostores @nanostores/solid
+  ```
+  </Fragment>
+  <Fragment slot="svelte">
+  ```shell
+  npm install nanostores
+  ```
+  :::note
+  Pas de package d'assistance ici ! Nano Stores peuvent être utilisés comme Svelte stores standard
+
+  :::
+  </Fragment>
+  <Fragment slot="vue">
+  ```shell
+  npm install nanostores @nanostores/vue
+  ```
+  </Fragment>
+</UIFrameworkTabs>
+
+Vous pouvez aller directement dans le [Guide d'utilisation de Nano Stores](https://github.com/nanostores/nanostores#guide) à partir d'ici, ou suivez notre exemple ci-dessous!
+
+## Exemple d'utilisation - Menu déroulant pour un panier de ecommerce
+
+Disons que nous construisons une interface de ecommerce simple avec trois éléments interactifs :
+-Un formulaire de soumission "ajouter au panier"
+-Un menu déroulant du panier pour afficher les articles ajoutés
+-Une bascule de chariot
```

Comment:
I think we can improve this translation, the best I got is "Boutton de bascule du menu déroulant" :thinking:

Reply:
Would love a concrete suggestion here if you have one!